### PR TITLE
Add an action sampler

### PR DESCRIPTION
**Pull request by axlbrown**
An action sampler class is created to have a clearer picture of the generative model.

### CODE REVIEW

File: policies/gac/gac_networks.py [hunk 1]
```diff
@@ -101,6 +101,8 @@ def __call__(self, state, taus, actions=None):
         state_embedding = tf.expand_dims(self.state_embedding(state), 1)
         # batch x action dim x 400
         noise_embedding = self.noise_embedding(taus)
+        print(noise_embedding.shape)
```
Comment (gwbcho) [inline policies/gac/gac_networks.py]: Why are we printing this value here?

File: policies/gac/gac_networks.py [hunk 2]
```diff
@@ -101,6 +101,8 @@ def __call__(self, state, taus, actions=None):
...
+        print(self.action_dim)
```
Comment (gwbcho) [inline policies/gac/gac_networks.py]: Also, this branch may require you to rebase onto master before merging. There were some modifications I made in order to correct some bugs I discovered while testing.

File: tests/policies/policy_helpers/test_helpers.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import tensorflow as tf
+from policies.policy_helpers.helpers import ActionSampler
+from policies.gac.gac_networks import AutoRegressiveStochasticActor
+
+def test_ActionSampler():
```
Comment (gwbcho) [inline tests/policies/policy_helpers/test_helpers.py:5]: Not exactly standard, but, if it works, it works.

**Review by gwbcho — CHANGES_REQUESTED** [review]

File: policies/gac/gac_networks.py (hunk 1 above)
Comment (axlbrown) [reply policies/gac/gac_networks.py]: Forgot to delete debug print.

File: tests/policies/policy_helpers/test_helpers.py (hunk 3 above)
Comment (axlbrown) [reply tests/policies/policy_helpers/test_helpers.py:5]: Yeah. Definitely not standard. Just for me to see it runs and the tensor shapes match.

File: policies/gac/gac_networks.py (hunk 2 above)
Comment (axlbrown) [reply policies/gac/gac_networks.py]: Yeah. I think after I merged master into this branch, the prints are gone.

**Review by gwbcho — APPROVED** [review]

**Pull request merged**